osbs/build/build_requestv2.py

```diff
@@ -267,7 +267,8 @@ def render(self, validate=True):
         self.set_data_from_reactor_config()
 
         # Adjust triggers for custom base image
-        if self.template['spec'].get('triggers', []) and self.is_custom_base_image():
+        if (self.template['spec'].get('triggers', [])
+                and (self.is_custom_base_image() or self.is_from_scratch_image())):
```

Comment:
This change doesn't seem to have a corresponding unit test.

Reply:
Added